### PR TITLE
HistogramPanel: Generate .cue model

### LINKED ISSUE #53584: Generate model for Histogram panel

### PR DESCRIPTION
**Pull request by mdvictor**
**What this PR does / why we need it**:
Generate .cue model for the histogram panel. Part of epic #48761.

**Which issue(s) this PR fixes**:

Closes #53584

## TODO:

- [x] Add docs for properties

### CODE REVIEW

File: public/app/plugins/panel/histogram/models.cue [hunk 1]
```diff
@@ -28,12 +28,24 @@ Panel: thema.#Lineage & {
...
 						combine?:     bool
 					} @cuetsy(kind="interface")
 
-					PanelFieldConfig: ui.GraphFieldConfig & {} @cuetsy(kind="interface")
+					PanelFieldConfig: {
+						ui.HideableFieldConfig
+						// Controls line width of the bars.
+						lineWidth?:    int32 & >= 0 & <= 10 | *1
+						// Controls the fill opacity of the bars.
+						fillOpacity?:  int32 & >= 0 & <= 100 | *80
```
Comment (sdboyer) [inline public/app/plugins/panel/histogram/models.cue:44]: If these are both >= 0, why not:
```suggestion
						lineWidth?:    uint32 & <= 10 | *1
						// Controls the fill opacity of the bars.
						fillOpacity?:  uint32 & <= 100 | *80
```

**Review by sdboyer — CHANGES_REQUESTED** [review]
Generally good, just two things related to numeric types!

**Comment by mdvictor** [comment]
Thank you for the review, @sdboyer. The changes make sense. I modified as requested.

**Review by sdboyer — APPROVED** [review]

**Comment by sdboyer** [comment]
looks great!

**Pull request merged**